plugins/nominal-connection-checker/test/connection_checker_test_helper.mocha.js

```diff
@@ -199,58 +198,59 @@ export function clearTwoBlockTests() {
  * called at the end of a suite of two block tests.
  */
 export function runTwoBlockTests() {
+  /**
+   * Creates a function which returns the input of an outer block.
```

Comment:
Nit: Creates a function which creates a new block and returns its next connection or first input connection.